examples/android/src/main/java/com/radixdlt/sargon/android/MainActivity.kt

```diff
@@ -57,69 +56,69 @@ class MainActivity : ComponentActivity() {
 
 @Composable
 fun WalletContent(modifier: Modifier = Modifier, storage: SecureStorageDriver) {
-    var walletState: Wallet? by remember { mutableStateOf(null) }
```

Comment:
Didn't have time to make example app compatible with sargon os. I can do that if needed.